### ISSUE TITLE
What does ↓self mean?

### ISSUE THREAD

**Issue by oingbong** [realm/SwiftLint#3317]
**[ask]**
To solve the Unneeded NotificationCenter Removal Violation, I looked for related codes and looked at `↓self`.
I don't understand what this means.
What does it mean?

**[code]**
```
triggeringExamples: [
            Example("""
            class Foo {
                deinit {
                    NotificationCenter.default.removeObserver(↓self)
                }
            }
            """)
            ,
            Example("""
            class Foo {
                deinit {
                    NotificationCenter.default.removeObserver(↓self,
                                                              name: UITextView.textDidChangeNotification, object: nil)
                }
            }
            """)
        ]
```

**[source]**
https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/Lint/UnneededNotificationCenterRemovalRule.swift

**Comment by marcelofabri** [comment]
`↓` is just where the violation will be reported. This is used in our unit tests to make sure not only the violations are triggered, but they're triggered where we expect.
